src/pdfpc.vala

```diff
@@ -193,6 +194,10 @@ namespace pdfpc {
                 GLib.printerr("--notes option detected. Disable auto grouping.\n");
             }
 
+            if (Pdfpc.is_Wayland_backend() && !Options.wayland_workaround) {
+                GLib.printerr("If you have wrong scaling of the windows, consider using --wayland-workaround\n");
```

Comment:
I'd place this into the documentation rather than outputting it every time pdfpc is started.

Reply:
I know that the man page might be the place, but do you think people will read it if they encounter a problem?

Reply:
They won't, true. We could add an FAQ to SUPPORT.md, or stick with the current solution, I don't feel strongly about it.

Reply:
I vote for a FAQ. IMHO, repeated warnings will be annoying if non-relevant. One can implement a one-time warning mechanism, but it isn't worth the trouble...